recipes/resiprocate/all/patches/002-fixed-mysql-include-path.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/repro/MySqlDb.cxx b/repro/MySqlDb.cxx
+index c5b0f67..982fcb8 100644
+--- a/repro/MySqlDb.cxx
++++ b/repro/MySqlDb.cxx
+@@ -7,10 +7,10 @@
+
+ #ifdef USE_MYSQL
+
+-#ifdef WIN32
+-#include <errmsg.h>
+-#else 
++#if __has_include("mysql/errmsg.h")
+ #include <mysql/errmsg.h>
++#else
++#include <errmsg.h>
+ #endif
```

Comment:
You could just rename is `002-unvendor-includes.patch` do this since Conan will always put them in the write place

```suggestion
#include <errmsg.h>
```